libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTracker.java

```diff
@@ -83,6 +83,14 @@
         EDITOR_TAPPED_UNLINK, // Visual editor only
         EDITOR_TAPPED_UNORDERED_LIST, // Visual editor only
         ME_ACCESSED,
+        ME_GRAVATAR_TAPPED,
+        ME_GRAVATAR_TAPPED_TOOLTIP,
```

Comment:
Will change it. As a background, I had put it in like that to make it a bit more clear that it is a specialisation of the tapping event.